src/components/ProgressForm/ProgressHeader.tsx

```diff
@@ -1,15 +1,25 @@
-import styles from '@/components/ProgressForm/ProgressForm.module.scss';
+import { FC } from 'react';
+import styles from '@/components/standup/standupContainer.module.scss';
+import { progressHeaderProps } from '@/types/ProgressUpdates';
 
-function ProgressHeader() {
+const ProgressHeader: FC<progressHeaderProps> = ({
+    totalMissedUpdates,
+    updateType,
+}) => {
     return (
-        <header className={styles.banner}>
-            <p>
-                You have <span className={styles.mark}> 2 missed</span> Progress
-                Updates
+        <div className={styles.standupBanner}>
```

Comment:
standup banner sounds too specific to the standup feature could we name it to progress banner (our API is also named progress.)

Reply:
fixed